### ISSUE TITLE
`sicksync add` should choose different ports 

### ISSUE THREAD

**Issue by laggingreflex** [appnexus/sicksync#56]
`sicksync add` chooses the same default `websocketPort` for all projects which causes problems if two projects are run simultaneously (#55). It should choose different ports based on what ports have already been used in other saved projects.

**Comment by joelgriffith** [comment]
So you can actually run two projects simultaneously with `sicksync start project1 project2 project3`. They all make use of the same underlying WS client, I'll try and document this somewhere more upfront.

**Comment by laggingreflex** [comment]
I haven't tested this yet, but do you mean that syntax actually makes it work? As opposed to doing `sicksync start project1`, `sicksync start project2`, `sicksync start project3` all separately and independently?

If so, independence would still be preferable IMO. Sometimes I like to start and stop one sync job while another one keeps running in the background.

**Comment by joelgriffith** [comment]
I think we can probably support both cases... I'll revisit this once the modernization effort is through

**Comment by laggingreflex** [comment]
This is still an issue. I haven't yet tried the syntax you suggested: `sicksync start project1 project2`, I'll try to later, but this is definitely an issue when starting projects separately. 

And I think starting projects separately should be given more importance, because otherwise you can't start syncing others projects without messing up previously running syncs, or you'd have to start them all again, if I'm understand it correctly.  

Separate ports might actually be a requirement in that case, as there are going to be separate processes. Although I'm not sure. 

In the meantime I'm manually making ports different to make my use case work.

**Comment by laggingreflex** [comment]
I've tried the single command syntax now: `sicksync start project1 project2` 

It has the problem I suspected. After starting syncs for two projects this way, and changing a file in one of the projects gets mistaken for that filename in the other project, giving the error like `ENOENT: no such file C:\proj-1\file-from-proj-2` because the corresponding file doesn't exist. 

So it seems they do need to be in separate ports?
